### DISCUSSION TITLE
Make note size, visible and it movement

### DISCUSSION THREAD

**Discussion by WakiN13** [CodenameCrew/CodenameEngine#326] [Ideas]
You can size it for making more characters in battle, cuz without size 3 characters dont have enough spaces for arrows, and visible for knew where are your arrows or where are arrows of opponent arrows, also movement of arrows can be usable in hard modes like in Dave n Bambi (its crazy).

**Comment by charlesisfeline** [comment]
you can do it in the chart editor but movement idk
